src/components/RowActions/RowActions.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import {Ellipsis} from '@gravity-ui/icons';
+import type {PopupPlacement} from '@gravity-ui/uikit';
+import {Button, Icon, Menu, Popup, useUniqId} from '@gravity-ui/uikit';
+
+import {block} from '../../utils';
+
+import i18n from './i18n';
+import type {TableActionConfig, TableActionGroup, TableActionsSettings} from './types';
+
+import './RowActions.scss';
+
+type RowActionsProps<I> = Pick<TableActionsSettings<I>, 'getRowActions' | 'rowActionsSize'> & {
+    item: I;
+    index: number;
+};
+
+const b = block('row-actions');
+const bPopup = block('row-actions-popup');
```

Comment:
Why do you declare the second class here? One component should have only one class.

Reply:
Left only 'row-actions'

Reply:
No, I'm not about that, i'm about a single block per component, you can use `b('popup')` for the popup